decidim/decidim
# Redesign: fix eye-icon on password fields when there's an error

#### :tophat: What? Why?
The eye-icon which hides your password in the form when confirming your password is incorrect was not adjusted properly. A change to ```bottom``` element in button allows for the icon to be centered properly.

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #11789 

#### Testing
1. Login 
2. Head over to account
3. Find change your password
4. In the fields asking to confirm your "new" and "current", make sure they're both different from each other.
5. Click change
6. See icon stay centred in the field box.

### :camera: Screenshots

<img width="801" alt="Screenshot 2023-10-23 at 14 26 14" src="https://github.com/decidim/decidim/assets/101816158/6bac890d-c9f9-4790-8ed6-ce9daa88302d">


:hearts: Thank you!
